pom.xml

```diff
@@ -3,6 +3,7 @@
 	<groupId>edu.wisc.umark</groupId>
 	<artifactId>uw-ui-toolkit</artifactId>
 	<version>0.3.2-SNAPSHOT</version>
+    <packaging>jar</packaging>
```

Comment:
how nice, indeed.  I guess having it here, doesn't hurt anything, just makes it even more explicit??...